docs/how-to/uc-boot-parameters.rst

```diff
@@ -59,6 +61,12 @@ To undo the modifications made to the kernel parameters, use the `snap unset`_ c
 
     snap unset system system.kernel.dangerous-cmdline-append
 
+---
+
+Once you are satisfied with the configurations, consider configuring them statically so that they are built into the image. 
+Read more on the `image creation guide <../uc-image-creation#custom-ubuntu-core-image>`_.
+
+.. Is there a better way to reference an anchor on another doc? 
```

Comment:
@AnneCYH do you have any idea about the link checker error:
```
(how-to/uc-boot-parameters: line   66) broken    ../uc-image-creation#custom-ubuntu-core-image -
```

The relative link works fine on the rendered docs.

Reply:
I've removed it for now: https://github.com/canonical/real-time-ubuntu-docs/pull/56/commits/f567019b68a8584f0048f8629bc805d76f85e212